include/ofi_mr.h

```diff
@@ -177,6 +178,10 @@ void ofi_memhooks_cleanup(void);
 
 extern struct ofi_mem_monitor *memhooks_monitor;
 
+/* Cuda memory monitor. */
+extern struct ofi_mem_monitor *cuda_monitor;
+void cuda_monitor_init(void);
+void cuda_monitor_teardown(void);
```

Comment:
Please call cleanup to align with rest of calls.

Reply:
Will do.